CHANGELOG.md

```diff
@@ -8,12 +8,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 - .NET 6 target with support for trimming (#1243, #1240)
+- Added DocumentFormat.OpenXml.Office.SpreadSheetML.Y2022.PivotRichData namespace
+- Added DocumentFormat.OpenXml.Office.PowerPoint.Y2019.Main.Command namespace
+- Added DocumentFormat.OpenXml.Office.PowerPoint.Y2022.Main.Command namespace
+- Added Child RichDataPivotCacheGuid to DocumentFormat.OpenXml.Office2010.Excel.PivotCacheDefinition
 
 ### Fixed
 - Removed reflection usage where possible (#1240)
 - Fixed issue where some URIs might be changed when cloning or creating copy (#1234)
 - Fixed issue in FlatOpc generation that would not read the full stream on .NET 6+ (#1232)
 - Fixed issue where restored relationships wouldn't load correctly (#1207)
+- Fixed .NET Core Remote Code Execution Vulnerability (#1222)
```

Comment:
This is not true - this was in a test dependency, not anything we shipped externally

Reply:
removed